### ISSUE TITLE
WakeWord is working but no output

### ISSUE THREAD

**Issue by GYJohn** [alexa/alexa-avs-sample-app#1116]
Hello,

the wakeword is working.
If i say alexa the following output apears:
`INFO:===> Connected to AVS client <===
INFO:===> WakeWordAgent: wake word detected <===
INFO:WakeWordAgent: State set to WAKE_WORD_DETECTED(3)
INFO:WakeWordAgent: State set to SENT_WAKE_WORD_DETECTED(4)
INFO:WakeWordAgent: IPC Command received:3
INFO:WakeWordAgent: State set to WAKE_WORD_PAUSE_REQUESTED(5)
INFO:SensoryWakeWordEngine: handling pause
INFO: *** THREAD JOINING: Sensory ***
INFO:SensoryWakeWordEngine: mainLoop thread ended
INFO:WakeWordAgent: State set to WAKE_WORD_PAUSED(6)
INFO:WakeWordAgent: IPC Command received:4
INFO:WakeWordAgent: State set to WAKE_WORD_RESUME_REQUESTED(7)
INFO:SensoryWakeWordEngine: handling resume
INFO:SensoryWakeWordEngine: mainLoop thread started
INFO:WakeWordAgent: State set to IDLE(2)
`

So my micro works. My output is working too.
Tested it with:
#533 
`
    For the audio out run: speaker-test -c2 -twav
    For microphone run:
        arecord -f S16_LE -Dplug:default test.wav
        aplay test.wav
`

Any ideas?

Thank you!

**Comment by frozenberg** [comment]
Hi @GYJohn,

Can I have a few more details about your setup? Are you on pi? If so, are you trying to run audio out of the 3.5mm jack or hdmi?

Thanks,
Fede

**Comment by GYJohn** [comment]
Hello @frozenberg,

yeah i am running the setup on a pi.
At first i was running the audio output out a 3.5mm jack and the microphone is a headset.

At the moment i am trying it with the headset as output and input. 
asound.conf:
`pcm.!default {
    type hw
    card 0
}

ctl.!default {
    type hw
    card 1
}
` 

I get sound and micro works.

**Comment by frozenberg** [comment]
Could you try the solution posted here: https://github.com/alexa/alexa-avs-sample-app/issues/689

Specifically:

disable the default audio device by commenting out
#dtparam=audio=on
in /boot/config.txt

Take back up of following files if they exists or Create new if doesn't exists
/home/pi/.asoundrc and /etc/asound.conf
with below content for using the USB sound card for both speaker and mic.

pcm.!default { type asym playback.pcm { type plug slave.pcm "plughw:1,0" format S16_LE } capture.pcm { type plug slave.pcm "plughw:1,0" }}

Reboot and check /home/pi/.asoundrc for only the above lines , if anything other is there remove and save the file then :
sudo /etc/init.d/alsa-utils restart

**Comment by GYJohn** [comment]
@frozenberg 

I made the config.txt stap.

I added the .asoundrc & asound.conf stuff.
I checked the .asoundrc after the reboot, everything like before.

The wakeword ist still working.
I cant hear any output out of the headset.

BUT i can the my question which i asked alexa on the alexa app!

**Comment by frozenberg** [comment]
Ok, let's try listing all your devices.

Run `aplay -l` and `arecord -l`.

In your case, the audio capture device and audio playback device are one and the same, right? (the headset) This might be causing an issue.

**Comment by GYJohn** [comment]
> root@raspberrypi:~# aplay -l
> **** Liste der Hardware-Geräte (PLAYBACK) ****
> Karte 1: Headset [Logitech USB Headset], Gerät 0: USB Audio [USB Audio]
>   Sub-Geräte: 1/1
>   Sub-Gerät #0: subdevice #0

> root@raspberrypi:~# arecord -l
> **** Liste der Hardware-Geräte (CAPTURE) ****
> Karte 1: Headset [Logitech USB Headset], Gerät 0: USB Audio [USB Audio]
>   Sub-Geräte: 0/1
>   Sub-Gerät #0: subdevice #0

Yeah but at first i had the 3.5 jack and the headset as input. Same issue

**Comment by frozenberg** [comment]
Can I see the .asoundrc you had before making any changes?

**Comment by GYJohn** [comment]
You mean before i edited your suggestion?

It was:

> pcm.!default {
>          type asym
>          playback.pcm {
>                  type plug
>                  slave.pcm "hw:0,0"
>          }
>          capture.pcm {
>                  type plug
>                  slave.pcm "hw:1,0"
>          } 
>  }
> 
>  ctl.!default {
>         type hw
>         card 0
> }

**Comment by frozenberg** [comment]
A few more questions:

What OS are you running?
What model Raspberry Pi are you working with?
Is your headset plugged in via USB and 3.5mm jack at the same time? There is a bit of confusion as to how your headset is set up.

**Comment by GYJohn** [comment]
Good evening @frozenberg 

I am using the rasbian Stretch.
It's model 3 B.
Yeah it's plugged in viva USB and my output is plugged in viva 3.5 jack too.
Should i try it without the output 3.5?

I am still using ur config:
pcm.!default { type asym playback.pcm { type plug slave.pcm "plughw:1,0" format S16_LE } capture.pcm { type plug slave.pcm "plughw:1,0" }}

So i don't think the 3.5 jack is the problem

**Comment by GYJohn** [comment]
At the moment i am using this config:
`pcm.!default { type asym playback.pcm { type plug slave.pcm "plughw:0,0" format S16_LE } capture.pcm { type plug slave.pcm "plughw:1,0" }}`

And enabled 
> #dtparam=audio=on
> in /boot/config.txt
again

I can use my Headset as input and my aux box as output.
Alexa still dont wanna answer :b

**Comment by GYJohn** [comment]
I bougth a little usb micro.
Still not working with any config.

**Comment by celinval** [comment]
Hi @GYJohn,

From the results of your aplay -l and arecord -l, it looks like you have the USB device configured to perform recording and playback. Are you able to play any audio in your headphones right now?

Can you please try to run the following:

`/proc/asound/cards`

and try to play some sound in your headphones. You can run

`aplay <some_wav_file>`

You can usually find some .wav files inside /usr/share/sounds/alsa.
